packages/plugin-react-navigation/test/react-navigation.test.tsx

```diff
@@ -0,0 +1,167 @@
+import Plugin from '../'
+import Client from '@bugsnag/core/client'
+import TestRenderer from 'react-test-renderer'
+import * as React from 'react'
+import { NavigationContainer, NavigationContainerRef } from '@react-navigation/native'
+
+jest.mock('@react-navigation/native')
+
+afterEach(() => jest.clearAllMocks())
+
+describe('plugin: react navigation', () => {
```

Comment:
It'd be good to have a test where we don't pass `ref`, so the plugin creates one itself

Reply:
The ref has to be passed in in order to mock the `getCurrentRoute()` behaviour of the navigation lib.

Reply:
Ah yeah; that makes sense